app/assets/javascripts/take-whole-pot-calculator.js

```diff
@@ -3,7 +3,7 @@
 
   var takeWholePotCalculator = {
     init: function() {
-      var $calculator = $('.js-take-whole-pot-calculator form');
+      var $calculator = $('.js-calculator--take-whole-pot form');
```

Comment:
Maybe worth doing that in another PR? I didn't want to step on the work to inline the calculators.